examples/models/llama/model.py

```diff
@@ -277,10 +287,12 @@ def __init__(self, **kwargs):
             self.model_ = prune_output_vocab(self.model_, output_prune_map)
 
     def get_eager_model(self) -> torch.nn.Module:
-        if self.dtype:
+        return self.model_
```

Comment:
is this a typo?

Reply:
Ah I was going to remove everything below and just have `return self.model_`